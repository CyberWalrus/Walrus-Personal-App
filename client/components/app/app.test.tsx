import * as Enzyme from "enzyme";
import { shallow } from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson, { Json } from "enzyme-to-json";
import * as React from "react";
import App from "./app";

Enzyme.configure({adapter: new Adapter()});
it(`App correctly render`, (): void => {
  const tree: Json = toJson(shallow(<App />));
  expect(tree).toMatchSnapshot();
});
