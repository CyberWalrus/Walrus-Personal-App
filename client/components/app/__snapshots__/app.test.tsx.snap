// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly render 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/test"
    />
    <Redirect
      to="/"
    />
  </Switch>
</BrowserRouter>
`;
